packages/synapse-sdk/src/test/rand.test.ts

```diff
@@ -1,6 +1,7 @@
 /* globals describe it */
+
+import { fallbackRandIndex, fallbackRandU256, randIndex, randU256 } from '@filoz/synapse-core/rand'
```

Comment:
@hugomrdias can you tell me about what your intention is for testing the components in synapse-core? I see it has no tests at the moment, are you intending to add some or are we supposed to be testing it all from the outside? I had to move `rand` into there but now the unit tests for it are sitting over here, awkwardly.